appwrite/website
# Add CCPA compliance to security documentation

## What does this PR do?

This PR updates the Appwrite security documentation. It adds information about CCPA compliance and the measures taken to protect user data. The new CCPA page provides details about Appwrite's compliance with the California Consumer Privacy Act.


## Test Plan

- Start up the website and navigate to `/docs/advanced/security/`. You should see the new CCPA card that's been added to the compliance section.
- Click the card or navigate to `/docs/advanced/security/ccpa`. You should see the new CCPA compliance page.

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

yes
